prisma/seed.ts

```diff
@@ -0,0 +1,101 @@
+import { faker } from "@faker-js/faker";
+import { PrismaClient } from "@prisma/client";
+import {
+  generateFakeDonation,
+  generateFakeDistributionsArray,
+  generateFakeDonor,
+  generateFakeTaxUnit,
+} from "./fakedata/generateFakedata";
+import { organizationsJSON } from "./fakedata/organizations";
+import { paymentsJSON } from "./fakedata/payments";
+import { distributionsJSON } from "./fakedata/distributions";
+
+const prisma = new PrismaClient();
+
+async function createFakeDonorWithFakeDonations() {
+  const fakeDonor = generateFakeDonor();
+
+  const fakeTaxUnit = generateFakeTaxUnit(fakeDonor.full_name);
+
+  const numberOfFakeDonations = faker.number.int({ min: 0, max: 10 });
+  const arrayOfFakeDonations = Array.from({ length: numberOfFakeDonations }).map(() =>
+    generateFakeDonation(),
+  );
+
+  const fakeDonorDB = await prisma.donors.create({
+    data: {
+      ...fakeDonor,
+      Donations: {
+        create: arrayOfFakeDonations,
+      },
+      Tax_unit: {
+        create: fakeTaxUnit,
+      },
+
+      //TODO: Virker som FundsDonations tabellen ikke brukes lengre
+      // Funds_donations: {
+      //   create: fakeFunds_donations()
+      // }
+    },
+    include: {
+      Donations: true,
+      Tax_unit: true,
+    },
+  });
+
+  fakeDonorDB.Donations.forEach(async (donation) => {
+    await createCombiningTables(donation, fakeDonorDB);
+  });
+}
+
+async function createCombiningTables(donation, fakeDonorDB) {
+  const KID = donation.KID_fordeling;
+
+  //TODO: Denne må med pga. foreign key constarint. Men stemmer denne FK-constarinten?
```

Comment:
AvtaleGire_replaced_distributions are created because of the Foreign Key constraint by in the [prisma.schema](https://github.com/stiftelsen-effekt/effekt-backend/blob/4c84dc6a1f6ea5d334e8960d475d8a47c19bb171/prisma/schema.prisma#LL69C1-L69C127)

Wondering if that FK constraint is necessary and could potentially be removed? Can't find it in the Prod Database.

If the FK should be optional, that seems to be an [issue with prisma](https://github.com/prisma/prisma/issues/18058)